### PR TITLE
add n8n installation tutorial

### PR DESCRIPTION
**Pull request by alexanderschau**
PR contains a tutorial to install [n8n](https://github.com/n8n-io/n8n) on a Netcup Webhosting instance.

I have read and understood the Contributor's Certificate of Origin at the end of the template and I hereby certify that I meet the contribution criteria described in it.

Signed-off-by: Alexander Schau <alexander@schau.email>

### CODE REVIEW

File: community-tutorials/workflow-automation-with-n8n-and-netcup-webhosting/01-en.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+Now that everything is in place, it is time to setup the final NodeJS configuration.
+
+To do this, go to **Websites & Domains** on the left side bar and select your domain. Then click **Node.js** and you will see a new page. Here click on the **Enable Node.js** button to active Node.
+
+Now that Node is running you can specify a Node.js Version. I recommend using the latest, but you must try which one works with n8n. Currently (November 2021) Node.js version 14 is working well.
+
+## Step 4.1 - Install NPM dependecies
+Now that Node is up and running, it's time to install all dependencies. You can do this by clicking the **NPM install** button at the top menu.
+
+If everything works fine, it should exit with some warnings, but we can ignore these.
```
Comment (raphi-0901) [inline community-tutorials/workflow-automation-with-n8n-and-netcup-webhosting/01-en.md:88]: Unfortunately I got some errors while installing the dependencies:

- /bin/sh: cmake: command not found
- make: *** [/var/www/vhosts/hosting164325.a2e22.netcup.net/httpdocs/n8n/node_modules/cpu-features/deps/cpu_features/build/Makefile] Error 127
- gyp ERR! build error
- gyp ERR! stack Error: `make` failed with exit code: 2
- gyp ERR! stack at ChildProcess.onExit (/opt/plesk/node/14/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
- gyp ERR! stack at ChildProcess.emit (events.js:400:28)
- gyp ERR! stack at Process.ChildProcess._handle.onexit (internal/child_process.js:282:12)
- gyp ERR! System Linux 4.9.0-16-amd64
- gyp ERR! command "/opt/plesk/node/14/bin/node" "/opt/plesk/node/14/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
- gyp ERR! cwd /var/www/vhosts/hosting164325.a2e22.netcup.net/httpdocs/n8n/node_modules/cpu-features
- gyp ERR! node -v v14.18.1
- gyp ERR! node-gyp -v v5.1.0
- gyp ERR! not ok

**Review by raphi-0901 — CHANGES_REQUESTED** [review]

File: community-tutorials/workflow-automation-with-n8n-and-netcup-webhosting/01-en.md (hunk 1 above)
Comment (alexanderschau) [reply community-tutorials/workflow-automation-with-n8n-and-netcup-webhosting/01-en.md]: It should work anyways. However i forgot to mention that the application script path needs to be changed (sorry for that :cry:). I updated the doc so everything should work fine now (please ignore this error, it has to do with plesk).

**Comment by raphi-0901** [comment]
Hey, thanks for updating. Unfortunately, I am getting an 403 Forbidden when I try reaching my Node-Application.

**Comment by raphi-0901** [comment]
Ok, it seems to work now. I just had to wait for a bit.

**Pull request merged**